modules/data.atmosphere/DESCRIPTION

```diff
@@ -57,12 +57,11 @@ Imports:
 Suggests:
     doParallel,
     foreach,
+    geonames,
+    nneo,
     parallel,
     progress,
     rlang (>= 0.2.0)
-Remotes:
-    github::ropensci/geonames,
-    github::ropenscilabs/nneo
```

Comment:
Right now the CRAN version of nneo is 0.1.0 and the dev version is 0.1.1.9313, but I don't know whether PEcAn needs any of the changes.

If keeping github version, note that repo has moved to `ropensci/nneo` (no "labs")